test/test_tensor_data.py

```diff
@@ -64,5 +64,12 @@ def test_data_float16(self):
     assert dat.shape == (2,2)
     # NOTE: python can't deref float16
 
+  def test_data_pathlib(self):
```

Comment:
This is not a good test for various reasons. It also doesn't test all the functionality

Reply:
:+1: I'll try again. What would be the ideal spot to put it in your opinion?

Reply:
http://www.catb.org/~esr/faqs/smart-questions.html

Reply:
Have you read all the other tests?

Reply:
Apparently didn't read enough of them. I'll go find a better spot.

Reply:
Moved to /test/test_tensor.py. Also made some changes to the test to improve cleanup. Based on the README of the CIFAR dataset, I moved the test to be more focused on numpy arrays that are stored in bin files. Hope this is more suitable. Hope this is an improvement, if not, please let me know. I would like to keep trying.